2021q3/syzkaller.adoc

```diff
@@ -0,0 +1,29 @@
+=== syzkaller on FreeBSD
+
+Contact: Mark Johnston <markj@FreeBSD.org>
+Contact: Michael Tuexen <tuexen@FreeBSD.org>
+
+syzkaller is a coverage-guided operating system kernel fuzzer.
+See the link:https://www.freebsd.org/status/report-2019-01-2019-03.html#Fuzzing-FreeBSD-with-syzkaller[syzkaller entry] in the 2019q1 quarterly report for an introduction to syzkaller.
+
+In the past quarter we made a concerted effort to shrink the backlog of reports from the public link:https://syzkaller.appspot.com/freebsd[syzbot] instance.
+A number of long-standing locking bugs in the socket subsystem have been fixed, and the SCTP protocol implementation has seen many bug fixes as well.
+Beyond that, link:https://github.com/freebsd/freebsd-src/search?o=desc&q=syzbot+OR+syzkaller&s=committer-date&type=commits[many bugs] in various other kernel subsystems have been fixed and the backlog has become substantially smaller over the past quarter.
+As a direct result of this effort, we have been able to identify regressions more easily and fix bugs closer to the time of introduction.
+Work is still ongoing to further shrink the backlog and get FreeBSD to a reasonably clean slate.
```

Comment:
I think you want "state" here, not "slate". (Or if there's an idiom I'm missing, perhaps reword for clarity.)

Reply:
"clean slate" is indeed an idiom. I just removed that part of the sentence since it doesn't add anything.